core/forwarder.py

```
@@ -1327,12 +1584,16 @@ def reset_refetch_miss_retry_state(channel: str, ids: list[int]) -> None:
                         error_types={},
                     )
             finally:
+                effective_strict_ack = False
                 if self._queue_clear_stale(clear_generation):
                     logger.info("[Send] 队列清空期间丢弃本轮发送结果。")
                     return
```

Ruff (0.15.20)

[warning] `return` inside `finally` blocks cause exceptions to be silenced

(B012)